2014 Primary Election El Dorado test failure

Missing write-in candidates. Not available at precinct-level, but just [totals](http://www.edcgov.us/elections/election/sov/095.write-in.pdf)

```
E           AssertionError: El Dorado failed
E           assert {('"Bo" Bogda...'): 7471, ...} == {('"Bo" Bogdan...'): 7471, ...}
E             Omitting 65 identical items, use -v to show
E             Left contains more items:
E             {('Patrick D. Hogan', 'El Dorado', 5.0, 'State Assembly'): 10}
E             Use -v to get the full diff
```

--- a/2014/20140603__ca__primary__el_dorado__precinct.csv
+++ b/2014/20140603__ca__primary__el_dorado__precinct.csv
@@ -5340,3 +5340,4 @@ El Dorado,96U40111,Treasurer,,DEM,John Chiang,84
 El Dorado,96U40111,U.S. House,4,REP,"Arthur ""Art"" Moore",40
 El Dorado,96U40111,U.S. House,4,IND,Jeffrey D. Gerlach,33
 El Dorado,96U40111,U.S. House,4,REP,Tom McClintock,122
+El Dorado,Write-In,State Assembly,5,LIB,Patrick D. Hogan,10

--- a/src/swdb/candidates/P14.json
+++ b/src/swdb/candidates/P14.json
@@ -155,5 +155,6 @@
  "Phill Tufi": "Phill A. Tufi",
  "KAREN JILL BERNAL": "Karen Jill Bernal",
  "JIMELLE L. WALLS": "Jimelle L. Walls",
- "NICKOLAS WILDSTAR": "Nickolas Wildstar"
+ "NICKOLAS WILDSTAR": "Nickolas Wildstar",
+ "PATRICK D. HOGAN": "Patrick D. Hogan"
 }

--- a/src/swdb/writeins.py
+++ b/src/swdb/writeins.py
@@ -114,7 +114,25 @@ def contra_costa_p14(_):
         house_df.to_dict(orient='records') + \
         assembly_df.to_dict(orient='records')
 
+
+def el_dorado_p14(candidate_norm):
+    writeins_url = 'https://www.edcgov.us/elections/election/sov/095.write-in.pdf'
+    writeins_req = requests.get(writeins_url)
+    writeins_req.raise_for_status()
+    writeins_pdf = PyPDF2.PdfFileReader(io.BytesIO(writeins_req.content))
+    cands = writeins_pdf.getPage(0).extractText()[162:180]
+    cand_split = [(cands[:-2], int(cands[-2:]))]
+    return [{'candidate': candidate_norm[c],
+             'county': 'El Dorado',
+             'office': 'State Assembly',
+             'district': '5',
+             'party': 'LIB',
+             'precinct': 'Write-In',
+             'votes': v} for
+            c, v in cand_split if v != 0]
+
 WRITEINS = {'P14': {'Alameda':   alameda_p14,
                     'Amador':    amador_p14,
                     'Calaveras': calaveras_p14,
-                    'Contra Costa': contra_costa_p14}}
+                    'Contra Costa': contra_costa_p14,
+                    'El Dorado': el_dorado_p14}}

